src/atoms/buttons/Button.js

```diff
@@ -8,34 +8,44 @@ function Button ( props ) {
     children,
     onClick,
     shake,
-    className
+    className,
+    buttonType,
+    disabled
   } = props;
 
   return (
-    <div
+    <button
       className={ classnames(className, styles[type], { [styles.shaking]: shake }) }
```

Comment:
@cisacke I believe that the className prop should be last in the arg list to allow for overrides.